docs/en/table_design/hybrid_table.md

```diff
@@ -2,42 +2,49 @@
 
 ## Comparisons between columnar storage and hybrid row-column storage 
 
 | **Storage format**         | **Storage method**                                           | **Scenarios**                                                |
 | -------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| Hybrid row-column storage | Data is stored in both row-by-row and column-by-column fashions. Simply put, a table that uses hybrid row-column storage contains an additional, hidden binary-type column `__row`. When data is written to the table, all values from the value columns of a row are encoded and written to the `__row` column (as shown below). As the data is stored in both row-by-row and column-by-column fashions, additional storage costs are incurred. ![img](../assets/table_design/hybrid_table.png) | <ul><li>Suitable for scenarios such as primary key-based point queries and partial column updates, because this storage method can help greatly reduce random IO and read-write amplification in these scenarios.</li><ul><li>Point queries, which are primary key-based simple queries that scan and return small amounts of data.</li><li>Queries against most or all of the fields from tables that consist of a small number of fields.</li><li>Partial column updates.</li><li>Prepared statements can be run on tables that use hybrid row-column storage, which can enhance query performance by saving the overhead of parsing SQL statements, and also prevent SQL injection attacks.</li></ul><li>Also suitable for complex data analysis.</li></ul>|
-| Column-oriented            | Data is stored in a column-by-column fashion ![img](../assets/table_design/hybrid_table.png) | <ul><li>Complex or ad-hoc queries on massive data. </li><li>Tables (such as wide tables) consist of many fields. Queries involve only a few columns. </li></ul>|
+| Hybrid row-column storage | Data is stored in both row-by-row and column-by-column fashions. Simply put, a table that uses hybrid row-column storage contains an additional, hidden binary-type column `__row`. When data is written to the table, all values from the value columns of a row are encoded and written to the `__row` column (as shown below). As the data is stored in both row-by-row and column-by-column fashions, additional storage costs are incurred. ![img](../assets/table_design/hybrid_table.png) | Suitable for use cases of both rowstore and columnstore storage xxx, but incurs additional storage costs.<ul><li>Use case of rowstore:</li><ul><li>High-concurrency point queries based on primary keys.</li><li>Queries against most fields from tables that consist of a small number of fields.</li><li>Partial column updates (more specifically, multiple columns and a small number of data rows need to be updated)</li></ul><li>Use case of columnstore: Complex data analysis.</li></ul>
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/9c9a4c1b-9c70-4f36-9886-c858f9ac9b2d)
what "xxx" is supposed to be?